docs/design/office-ui-fabric.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Office UI Fabric in Office Add-ins 
```

Comment:
@lindalu-MSFT I'll leave this article for @Rick-Kirkham to handle Fabric/Fluent situation